catalyst/dl/utils.py

```diff
@@ -53,7 +53,7 @@ def create_tflogger(logdir: str, name: str) -> SummaryWriter:
     @staticmethod
     def create_loggers(
         logdir: str, loaders: List[str]
-    ) -> 'OrderedDict[str, SummaryWriter]':
```

Comment:
i think, for types is better to put `'` quotes to note, that it is not a typical string